File: talleres_practicas/practica_parcial_2021-I/punto 2/punto 2.cpp
#include<iostream>
using namespace std;

void upDownCount();
void upDownCountWhile();

int main(){
	
   upDownCount();
   cout<<"\n\n\t";
   cout<<"\n\n\t";
   upDownCountWhile();
   cout<<"\n\n\t";	
   system("pause");
   return 0;
}


void upDownCount(){
	
	for(int oneToEnd = 1, endToOne=100; oneToEnd<=100 && endToOne>=1;oneToEnd++, endToOne--){
		cout<<"\n\t Count Up : "<<oneToEnd<<" | "<<" Count Down : "<<endToOne;
	}
}

void upDownCountWhile(){
	int oneToEnd = 1, endToOne=100;
	
	while(oneToEnd<=100 && endToOne>=1){
		
		cout<<"\n\t Count Up : "<<oneToEnd<<" | "<<" Count Down : "<<endToOne;
		oneToEnd++;
		endToOne--;	
	}
	
}

File: talleres_practicas/practica_parcial_2021-I/punto 2/punto 2.py
def upDownCountWhile():
  oneToEnd = 1
  endToOne=100

  while(oneToEnd <= 100 and endToOne >=1):
    print("\n\t Count Up : ",oneToEnd," | ",
    " Count Down : ",endToOne)

    oneToEnd += 1
    endToOne -= 1


def upDownCount():

  for oneToEnd, endToOne in zip(range(1, 101), range(100, 0, -1)):
    print("\n\t Count Up : ",oneToEnd," | ",
    " Count Down : ",endToOne)


### Function Main #####
def main():
  upDownCount()
  print("\n\n\t")
  upDownCountWhile()
  print("\n\n\t")

############# End of function ##################

###Execute Function main ######
if __name__ == '__main__':
  main()